packages/perseus/src/renderer.tsx

```diff
@@ -1696,8 +1687,11 @@ class Renderer extends React.Component<Props, State> {
 
     /**
      * Returns an array of the widget `.getUserInput()` results
+     *
+     * TODO: can we remove this?
+     * @deprecated use getUserInputMap
```

Comment:
I think we should try!  The return value here is an array that is very unfriendly (as we have no mapping back to widget ID for each entry).